src/actions/CryptoExchangeActions.js

```diff
@@ -154,56 +155,68 @@ export const shiftCryptoCurrency = () => async  (dispatch: any, getState: any) =
       dispatch(actions.dispatchAction(Constants.SHIFT_COMPLETE))
       console.log(broadcastedTransaction)
       console.log(savedTransaction)
-      setTimeout(() => {
-        Alert.alert(s.strings.exchange_succeeded, s.strings.exchanges_may_take_minutes)
-      },1)
+      setTimeout(() => { Alert.alert(s.strings.exchange_succeeded, s.strings.exchanges_may_take_minutes) }, 1)
     } catch (error) {
       dispatch(actions.dispatchActionString(Constants.SHIFT_ERROR,error.message))
-      setTimeout(() => {
-        Alert.alert(s.strings.exchange_failed, error.message)
-      },1)
+      setTimeout(() => { Alert.alert(s.strings.exchange_failed, error.message) }, 1)
```

Comment:
Please don't do these types of cleanup. These are harder to set breakpoints on and just make the code review more polluted.